include/provider/data/data_transaction.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ERD_PROXY_DATA_TRANSACTION_H
+#define ERD_PROXY_DATA_TRANSACTION_H
+
+#include <string>
+
+struct TransactionHash
```

Comment:
Perhaps add a comment which explains that the `hash` member is in hex?

Reply:
Added a comment for it